File: Composite Pattern/PHP/Man.php
<?php

require_once "Person.php";

class Man extends Person 
{
    public function __construct(int $age)
    {
        $this->gender = $this::class;
        $this->age = $age;
    }

    public function getInfo()
    {
        return "This is a " . $this->getAge() . " years old " . $this->getGender() . ".";
    }
}

?>

File: Composite Pattern/CS/Man.cs
namespace CompositePattern
{
    public class Man : Person
    {
        public Man(string name, int age)
        {
            this.Name = name;
            this.Age = age;
            this.Gender = "Male";
        }
        
        public override string GetInfo()
        {
            return "This is a " + this.GetGender();
        }
    }
}